cmd/analyze/analyze.go

```diff
@@ -53,50 +54,44 @@ var AnalyzeCmd = &cobra.Command{
 			os.Exit(1)
 		}
 
-		var aiClient ai.IAI
-		switch backendType {
-		case "openai":
-			aiClient = &ai.OpenAIClient{}
-			if err := aiClient.Configure(token, language); err != nil {
-				color.Red("Error: %v", err)
-				os.Exit(1)
-			}
-		default:
-			color.Red("Backend not supported")
+		aiClient := ai.NewClient("openai")
```

Comment:
Shouldn't we keep the backend variable here to maintain consistency with the rest of the code? From what I understand, regardless of the value sent in the backend variable, we currently return `openai`. I believe this will make the code easier to modify when we add a new backend.

Reply:
Thank you, yes, changed it :-)